context/app/static/js/theme.js

```diff
@@ -0,0 +1,79 @@
+import { createMuiTheme } from '@material-ui/core/styles';
```

Comment:
Can you add links to the corresponding style guide files? I wouldn't want to waste too much on it, but we'd want these to stay in sync, right?